### PR TITLE
API-1322 upgrade various core packages and fix breaking changes

### PR DESCRIPTION
**Pull request by stephennovosel**
### Description

Upgrades to various packages within the developer-portal and fixes to breaking changes associated to packages

### Requested feedback

Fixes #

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -9,7 +9,6 @@
     "allowJs": true,
     "jsx": "react",
     "moduleResolution": "node",
-    "rootDir": "src",
```
Comment (mjlumetta) [inline tsconfig.json:12]: why was this removed?

**Comment by stephennovosel** [comment]
@mjlumetta good catches, let me look over this and add those changes to the branch.

File: tsconfig.json (hunk 1 above)
Comment (stephennovosel) [reply tsconfig.json:12]: The jest upgrade started throwing an error for referring to modules used outside of the rootDir scope. Not 100% sure why, but this was able to remove that error without breaking anything else

**Comment by rtravitz** [comment]
PR is closing to perform upgrades separately and 🤞 reduce the complexity of each individual PR.
